contracts/starknet/tests/test_compute_engines/test_summary_stats.cairo

```diff
@@ -168,9 +209,9 @@ func test_volatility2{syscall_ptr: felt*, range_check_ptr}() {
     %{ ids.summary_stats_address = context.summary_stats_address %}
 
     let (_volatility) = ISummaryStats.calculate_volatility(
-        summary_stats_address, 1, 1664805721, 1664806093
+        summary_stats_address, 1, 1664805721, 1664806093, 10
     );
-    assert _volatility = 283158920430;  // returns value in decimals
+    assert _volatility = 5315031800;  // returns value in decimals
```

Comment:
Please add a comment on the math to derive the true value being used to benchmark the tests given that has changed

Reply:
wasn't broken, we updated the volatility to use `N-1` instead of `N` as a divisor for degrees of freedom. update from sara. she can provide context if you are curious.

Reply:
Used N-1 instead of N for calculating volatility as it is over a random sample. Also, the coefficient wasn’t based on standard units (annualized), so that was also fixed.